main.py

```diff
@@ -115,6 +115,15 @@ def get_redirect_route(regex_route, handler, defaults=None):
 
     Warning: this method strips off parameters after the trailing slash. URLs
     with parameters should be formulated without the trailing slash.
+
+    Args:
+        regex_route: unicode. A raw string representing a route.
+        handler: BaseHandler. A callable to handle the route.
+        defaults: NoneType. Default or keywords to be passed into a
```

Comment:
Hmm -- not sure about NoneType here. I agree that one possible value of defaults is None, but it might not be the only possible value. Can you check the webapp2 documentation to see what is actually needed here?

Reply:
The type is of dict I made the changes.

Reply:
Done!